tool/site/client-react/src/components/selectField.tsx

```diff
@@ -0,0 +1,30 @@
+import React, {FC} from "react"
+import {Select} from "nhsuk-react-components"
+import {Field} from "formik"
+import {Coding} from "fhir/r4"
+
+export interface SelectFieldProps {
+  name: string
+  label: string
+  fieldOptions: Array<Option>
+}
+
+interface Option {
```

Comment:
Could use `code` and `display` instead of `value` and `text`. I think those names are clearer, plus it would remove the need for the `convertCodingsToOptions()` function as a `Coding` already conforms to that interface.

Reply:
i didnt want to tie selectField to a Coding so left it open for someone to implement their own method if they have a different data type

Reply:
Yeah I'm not suggesting tying it to `Coding`, I'm suggesting that we should use `code` and `display` instead of `value` and `text`.